chef-utils/README.md

```diff
@@ -0,0 +1,168 @@
+# Chef Utils gem
+
+**Umbrella Project**: [Chef Infra](https://github.com/chef/chef-oss-practices/blob/master/projects/chef-infra.md)
+
+**Project State**: [Active](https://github.com/chef/chef-oss-practices/blob/master/repo-management/repo-states.md#active)
+
+**Issues [Response Time Maximum](https://github.com/chef/chef-oss-practices/blob/master/repo-management/repo-states.md)**: 14 days
+
+**Pull Request [Response Time Maximum](https://github.com/chef/chef-oss-practices/blob/master/repo-management/repo-states.md)**: 14 days
+
+## Getting Started
+
+Chef Utils gem is common code and mixins for the core Chef Infra ruby gems.
+
+### Platform Helpers
+
+Individual platforms for when code MUST be different on a case-by-case basis.  It is generally encouraged to not use these and to use
+the `platform_family` helpers unless it is known that code must be special cased for individual platforms.
+
+* `centos?`
+* `clearos?`
+* `linuxmint?`
+* `nexentacore?`
+* `omnios?`
+* `openindiana?`
+* `opensolaris?`
+* `opensuse?`
+* `oracle?`
+* `raspbian?`
+* `redhat?`
+* `scientific?`
+* `ubuntu?`
+
+When the platform name matches a platform family name, the platform-specific helper gains a `_platform?` suffix (e.g. the debian platform family is
+`debian?` while the debian platform is `debian_platform?`).  Many of these are included simply for API consistency and completeness, where there
+is only a single platform in the platform family the platform family helper should be preferred instead (e.g. always use `amazon?` rather than
+`amazon_platform?`).
+
+* `aix_platform?`
+* `amazon_platform?`
+* `arch_platform?`
+* `debian_platform?`
+* `dragonfly_platform?`
+* `fedora_platform?`
+* `freebsd_platform?`
+* `gentoo_platform?`
+* `mac_os_x_platform?`
+* `netbsd_platform?`
+* `openbsd_platform?`
+* `slackware_platform?`
+* `smartos_platform?`
+* `solaris2_platform?`
+* `suse_platform?`
+* `windows_platform?`
+
+### Platform Family Helpers
+
+These should be the most commonly used helpers to identify the type of node which group somewhat similar or nearly identical types of platforms.
+There are helpers here which are also meta-families which group together multiple types into supertypes.
+
+* `aix?`
+* `amazon?`
+* `arch?`
+* `debian?`
+* `dragonflybsd?`
+* `fedora?`
+* `freebsd?`
+* `gentoo?`
+* `mac_os_x?`
+* `netbsd?`
+* `openbsd?`
+* `rhel?`
+* `smartos?`
+* `solaris2?`
+* `suse?`
+* `windows?`
+* `windows_ruby_platform?` - this is always true if the ruby VM is running on a windows host and is not stubbed by chefspec
+
+Super Families:
+
+* `redhat_based?` - rhel, fedora and derivaties (including pidora, but not including amazon)
```

Comment:
this should actually include amazon now

Reply:
that turns it into `fedora_derived?`.

i could nuke this one, because it was something that either you or thom wanted and i couldn't see the point of, but we had a bit argument around it in the RFC.

i've used `fedora_derived` internally in the core chef changes, but nothing required `redhat_based` at all.